### DISCUSSION TITLE
How to access image uploaded to tmp path from edge?

### DISCUSSION THREAD

**Discussion by subhamcodes** [adonisjs/core#1291] [Help] [answered]
In adonis 4.1 I used to get the tmpPath by using Helpers. However, I have not found anything similar in adonis 5.
Adonis 4.1 example given below.

    Helpers.tmpPath('/uploads/xyz.png')

**Comment by thetutlage** [comment, marked as answer]
You can get it using `Application`. I understand, its not documented.

```ts
import Application from '@ioc:Adonis/Core/Application'

Application.tmpPath
```

For rest of the methods, you can rely on Typescript intellisense 😄

**Reply by subhamcodes** [reply]
Thanks its working.

**Reply by mcordoba** [reply]
God... I have been looking for it. It should be documented please.
